PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -483,7 +482,7 @@ INT_PTR CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM)
 // return true if one of CommandShortcuts is deleted. Otherwise false.
 void Accelerator::updateShortcuts() 
 {
-	vector<int> incrFindAccIds;
+	vector<UINT> incrFindAccIds;

```

Comment:
Could you please clarify implications of it?
With your suggested fix type casting is also needed (UINT -> ULONG for all int values that added to incrFindAccIds) plus more memory will be used in the end.

I write small code snippet to see how type conversion look and how it affects assembly code :
```C++
	unsigned int s1 = 1, s2 = 2;
00174BAE  mov         dword ptr [s1],1  
00174BB5  mov         dword ptr [s2],2  
	unsigned long l1 = 1000000000l;
00174BBC  mov         dword ptr [l1],3B9ACA00h  

	if (s1 == s2) //UINT AND UINT
00174BC3  mov         eax,dword ptr [s1]  
00174BC6  cmp         eax,dword ptr [s2]  
00174BC9  jne         main+52h (0174BE2h)  
//...
	if (s1 == l1) //UINT AND ULONG
00174BE2  mov         eax,dword ptr [s1]  
00174BE5  cmp         eax,dword ptr [l1]  
00174BE8  jne         main+71h (0174C01h)  
//...

	if (l1 == s1) //ULONG AND UINT
00174C01  mov         eax,dword ptr [l1]  
00174C04  cmp         eax,dword ptr [s1]  
00174C07  jne         main+90h (0174C20h)  
//...	


	if (l1 == s1) // first bigger type
00174C01  mov         eax,dword ptr [l1]  
00174C04  cmp         eax,dword ptr [s1]  
00174C07  jne         main+90h (0174C20h)  
//...	

```
I see no difference whatsoever - so it looks to me we shouldn't bother about safe implicit built-in type conversions.